apps/emqx_bridge_datalayers/test/emqx_bridge_datalayers_SUITE.erl

```diff
@@ -1045,6 +1046,8 @@ t_write_failure(Config) ->
     ),
     ok.
 
+%% XXX: this required the previous test to be run first.
+%% otherwise the table `mqtt` won't be created.
```

Comment:
We shouldn't depend on a specific order tests are executed.  Better to do the required setup/teardown for each test case as appropriate.

Reply:
marked a TODO to fix the flaky test.
[EMQX-14576](https://emqx.atlassian.net/browse/EMQX-14576)